dev/format.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
```

Comment:
Just curious, how's this better `cd ..`?

Reply:
@sooheon the part

```suggestion
cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
```

ensures that regardless of from where the bash script is called we end up in the scripts directory and than

```suggestion
/../
```

goes one level upwards.

I couldn't find the original source  on Stackoverflow which inspired this years ago but hear: https://stackoverflow.com/a/39340259/1087911 a user explains it perfectly.

If you use `cd ..` you might end up somewhere relative to your current working directory instead cd'ing relative to the bash script.